setup.py

```diff
@@ -103,7 +103,7 @@ def setup_requirements() -> Tuple[List[str], List[str], List[str]]:
     # Framework-specific requirements
     if not bool(int(os.getenv("NVTE_RELEASE_BUILD", "0"))):
         if "pytorch" in frameworks:
-            install_reqs.extend(["torch"])
+            install_reqs.extend(["torch>=2"])
```

Comment:
Should we make sure to avoid PyTorch 2.0?
```suggestion
            install_reqs.extend(["torch>=2.1"])
```